xchange-bithumb/src/main/java/org/knowm/xchange/bithumb/dto/trade/BithumbTradeResponse.java

```diff
@@ -1,82 +1,21 @@
 package org.knowm.xchange.bithumb.dto.trade;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import java.math.BigDecimal;
-import java.util.List;
 import org.knowm.xchange.bithumb.dto.BithumbResponse;
 
-public class BithumbTradeResponse extends BithumbResponse<List<BithumbTradeResponse.BithumbTrade>> {
+public class BithumbTradeResponse extends BithumbResponse {
```

Comment:
if this does not have the data field then it should not extend from `BithumbResponse`

besides it this now does not return a list of trades then there is no sense in calling it a Trade response

Reply:
This is only used to return the orderId of newly placed orders, so it is the REST response to place, marketBuy, marketsell.
I have removed dependency on BithumbResponse